helm/dagster/schema/schema_tests/test_user_deployments.py

```diff
@@ -504,7 +504,7 @@ def test_user_deployment_default_image_tag_is_chart_version(
     assert image_tag == chart_version
 
 
-@pytest.mark.parametrize("tag", [5176135, "abc1234"])
+@pytest.mark.parametrize("tag", [5176135, "abc1234", "20220531.1", "1234"])
```

Comment:
is non-string float worth including / caring out? (20220531.1 without the quotes)

Reply:
converting floats to strings in Helm is basically a forsaken path - see the root issue in the summary.

In Helm, the float gets parsed into scientific notation if we convert the float to string. At that point, we should just recommend to the user to cast their float as a string (as now, we can account for that)